layers/+tools/habitrpg/keybindings.el

```diff
@@ -0,0 +1,3 @@
+;; Key Bindings for habitrpg.el
+(spacemacs/set-leader-keys "aH" 'habitrpg-status)
+(add-hook 'org-after-todo-state-change-hook 'habitrpg-add 'append)
```

Comment:
These should be in `packages.el` as part of the init function to which they belong.